### PR TITLE
Remove some membership cruft from the README

### PR DESCRIPTION
**Pull request by philwills**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,8 +131,10 @@ S3 bucket, and if private credentials need adding or updating, they need to be u
 
 You can download and update credentials like this
 
-    aws s3 cp s3://contributions-private/DEV/contributions.private.conf /etc/gu
-    aws s3 cp /etc/gu/contributions.private.conf s3://contributions-private/DEV/
+```
+sudo aws s3 cp s3://contributions-private/DEV/contributions.private.conf /etc/gu/ --profile membership
```
Comment (alexduf) [inline README.md:135]: I think it's probably better practice to leave the `sudo` out, and let the developer handle the file. Typically there's no need to run `aws` as root if you can manually move the file afterwards (But that's me being paranoid and pedantic, I'll go put my tinfoil hat now if you don't mind)

**Review by alexduf — APPROVED** [review]
Please feel free to disregard previous comment and merge 👍

File: README.md (hunk 1 above)
Comment (AWare) [reply README.md:135]: We could be good and use ~/.gu instead?

Comment (alexduf) [reply README.md:135]: yeah personally I put everything in ~/.gu, and create symlinks when there's no other choice (nginx for instance)

Comment (philwills) [reply README.md:135]: Yea, I don't like dumping anything accessible only by `sudo`.  I'll probably do a separate PR around that. Once I have something working and can confirm that putting things in `~/.gu` works fine too.

**Pull request merged**